Wox.Infrastructure/StringMatcher.cs

```diff
@@ -8,14 +8,16 @@ namespace Wox.Infrastructure
 {
     public static class StringMatcher
     {
+        public static MatchOption EmptyMatchOption = new MatchOption();
```

Comment:
this wouldn't be necessary, we can just switch to using the other FuzzySearch constructor